packages/react/src/internal/useMatchMedia.ts

```diff
@@ -8,39 +8,46 @@
 import { useState, useEffect } from 'react';
 import { canUseDOM } from './environment';
 
-export function useMatchMedia(mediaQueryString) {
+// TODO: When can support for `addListener` be removed?
```

Comment:
I think it's safe to remove 👍 

`addEventListener` already had near-full browser support as of the v11 release (March 2021). We probably could have removed it then but didn't. The only thing was the `signals` property wasn't supported until Safari 15, which was September 2021. https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#browser_compatibility

Not a huge deal and this is an internal-only helper so no concerns about backwards incompatibility.